src/signals/incident-management/containers/IncidentDetail/index.js

```diff
@@ -199,19 +184,18 @@ export class IncidentDetail extends React.Component {
                     )}
 
                     {previewState === 'showLocation' && (
-                      <LocationPreview
-                        location={incident.location}
-                        onEditLocation={this.onEditLocation}
-                      />
+                      <LocationPreview location={incident.location} onEditLocation={this.onEditLocation} />
                     )}
 
                     {previewState === 'editLocation' && (
-                      <LocationForm
-                        incidentId={incident.id}
-                        location={incident.location}
-                        onPatchIncident={onPatchIncident}
-                        onClose={this.onCloseAll}
-                      />
+                      <MapContext>
```

Comment:
Isn't it possible to wrap the `LocationForm` component's `render()` function return value with `MapContext`? Seems a bit odd to have to wrap at at this level.

Reply:
Good point. I'll change that.